README.md

```diff
@@ -139,10 +139,11 @@ In order to provide your own repository configurations, you are required to set
 of repository resource type configurations in the following format:
 
 ```
-repository_resource_type: #ex... 'apt::source` or `apt::pin` or `yumrepo`
-  repository_resource_title:
-    repository_resource_attribute1: 'value'
-    repository_resource_attribute2: 'value'
+gitlab::repository_configuration:
```

Comment:
can you also add `yaml` behind the backticks, so we get proper syntax highlighting?

Reply:
good call, will do!